doc/usage.rst

```diff
@@ -294,6 +302,11 @@ The table is composed of one row per reservation and 5 columns:
 #. The start time of this reservation
 #. The end time of this reservation
 
+This view, like Jobs View, can be applied with Slurm's Private Data as well.
+If ``reservations`` has been defined previously in Slurm configuration, regular
+users are prevented from viewing others' reservations. Only superusers are
+allowed to consult all the reservations.
+
 Gantt view
 ----------
 
```

Comment:
I guess the feature also impacts the jobsmap, 3d and gantt view since they all call `jobs/`, doesn't it? If yes, update the doc accordingly.